main.py

```diff
@@ -60,8 +57,14 @@ def get_edit(request: Request, item_id: int, db: Session = Depends(get_db)):
 
```

Comment:
Consider adding validation for the `content` and `notes` fields to ensure data quality.